docs/docs/references/core/configuration.md

```diff
@@ -19,15 +19,18 @@ If you run Logto via `npm start` in the project root, `NODE_ENV` will always be
+| ADMIN_PORT              | `3002`                               | `number`                                                 | The local port that Logto Admin Console listens to.                                                                                                                                                                      |
+| ADMIN_DISABLE_LOCALHOST | N/A                                  | <code>string &#124; boolean &#124; number</code>         | Set it to `1` or `true` to close the port for Admin Console. With `ADMIN_ENDPOINT` unset, it'll completely close the Admin Console.                                                                                      |
+| DB_URL                  | N/A                                  | `string`                                                 | The [Postgres DSN](https://www.postgresql.org/docs/14/libpq-connect.html#id-1.7.3.8.3.6) for Logto database.                                                                                                             |
+| HTTPS_CERT_PATH         | `undefined`                          | <code>string &#124; undefined</code>                     | See [Enabling HTTPS](#enabling-https) for details.                                                                                                                                                                       |
+| HTTPS_KEY_PATH          | `undefined`                          | <code>string &#124; undefined</code>                     | Ditto.                                                                                                                                                                                                                   |
+| TRUST_PROXY_HEADER      | `false`                              | `boolean`                                                | Ditto.                                                                                                                                                                                                                   |
+| ENDPOINT                | `'protocol://localhost:$PORT'`       | `string`                                                 | You may specify a URL with your custom domain for online testing or production. This will also affect the value of the [OIDC issuer identifier](https://openid.net/specs/openid-connect-core-1_0.html#IssuerIdentifier). |
+| ADMIN_ENDPOINT          | `'protocol://localhost:$ADMIN_PORT'` | `string`                                                 | You may specify a URL with your custom domain for production (E.g. `ADMIN_ENDPOINT=https://admin.domain.com`). This will also affect the value of Admin Console Redirect URIs.                                           |
```

Comment:
Maybe we should also add `DISABLE_ADMIN_LOCALHOST`?

Reply:
added bruh